cvat-ui/src/components/resource-sorting-filtering/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './styles.scss';
+import SortingComponent from './sorting';
+import ResourceFilterHOC from './filtering';
+
+const defaultVisibility: {
+    predefined: boolean;
```

Comment:
Probably type definition is redundant there

Reply:
Removed.